### ISSUE TITLE
[docs] Need Minimal Splashscreen instructions

### ISSUE THREAD

**Issue by slimandslam** [tauri-apps/tauri-docs#2486]
The lab here is great:  https://v2.tauri.app/learn/splashscreen/
However, it is overly complicated as the vast majority of users will just want to 
do setup tasks in JS.   A basic example, that simply waits in JS as the task, is needed.

**Comment by simonhyll** [comment]
I mean, the lab does show how you do a JS based setup task as well. If you just replace the line that invokes a function in the backend for closing the window when all tasks are done with the JS API equivalent of closing the splashscreen then you have a lab for JS.

The idea with the labs is that you do them from start to finish as quick little throwaway projects for the sake of learning, where in the existing lab you learn how to do a setup task in both the frontend and the backend. I don't feel like it'd be an improvement to remove the backend related stuff for the sake of showing JS devs one line of code for closing the splashscreen (and opening the main window).

**Comment by slimandslam** [comment]
The lab goes from zero to complicated without showing the solution for
(likely) the vast majority of developers.  The way to greatly improve the
lab is to start with the simplest case. Using the existing frontend JS/TS
is fine, but then no backend task in Rust.  Show that solution and *then*
"improve" it by adding the code to do both backend and frontend tasks as
it's currently shown.

**Comment by slimandslam** [comment]
This scaled down version works on the Mac but not on mobile. I'm not sure how to get it to work on mobile. 

Line in main.ts:
```
// Set the frontend task as being completed
     invoke('close_splashscreen');
```

lib.rs:
```
use tauri::WebviewWindow;
use tauri::Manager;

#[tauri::command]
async fn close_splashscreen(window: WebviewWindow) {

      let splash_window = window.get_webview_window("splashscreen").unwrap();
        let main_window = window.get_webview_window("main").unwrap();
        splash_window.close().unwrap();
        main_window.show().unwrap();
}

#[cfg_attr(mobile, tauri::mobile_entry_point)]
pub fn run() {
  tauri::Builder::default()
    .invoke_handler(tauri::generate_handler![close_splashscreen])
    .run(tauri::generate_context!())
    .expect("error while running tauri application");

}

```

**Comment by FabianLars** [comment]
Mobile currently has no concept of multiple windows/webviews so you'd need a frontend-only solution like a different react-router route for the splashscreen.

**Comment by slimandslam** [comment]
Yes. I just tried the original splashscreen solution and it also fails on mobile.

**Comment by slimandslam** [comment]
So, iin summary, I  propose the following changes:

1. The lab should say, in the first few sentences, that it does not work on mobile giving the reason provided by @FabianLars above.  Note that the lab currently has the user answering the mobile setup question as "yes", for some reason:
```Would you like to setup the project for mobile as well? yes```
That should probably be changed to "no".  Better yet, provide a mobile-only solution as another lab. 

2. The lab should start off, using mostly the same code, but using the JS-focused  solution I have provided above (no Rust task).

3. The lab then proceeds with the current solution (no changes needed).

**Comment by slimandslam** [comment]
One more thing: At least in React, the splashscreen.html file needs to be in the public directory. Or in the "assets" or "static" directory, depending on which frontend framework you use. 
According to FabienLars, if you want the splashscreen.html file to be at the top-level, you can configure vite to handle multiple entrypoints as described here: https://vitejs.dev/guide/build.html#multi-page-app
